Google.PowerShell.Tests/Compute/ManagedGceInstanceCmdletTests.cs

```diff
@@ -148,5 +148,98 @@ public void TestRemoveGceManagedInstanceGroupPipelineZonal()
 
             instances.VerifyAll();
         }
+
+        /// <summary>
+        /// Runs PowerShell script script to create a GceManagedInstanceGroup.
+        /// Checks that the group created has name instanceGroupName.
+        /// </summary>
+        /// <param name="script"></param>
+        private void TestAddGceManagedInstanceGroupHelper(string script, string instanceGroupName)
```

Comment:
The name of this method doesn't say what it does, only where it is used. Part of the problem is it does not do one thing; it does two things that happen to be repeated and always next to each other.

The last three lines could conceivably be a good helper function named `AssertSingleGroupReturned`, but I'm not sure the logic is complicated enough to warrant it. I would err on the side of verbosity here and inline this function.

See http://go/unit-test-practices#assertions